src/utilities/externalLinks.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Central place for commonly used external links
+ * Esp for links that are used in multiple places and might change URL in the future
+ */
+
+export const externalLinks = {
+  give: 'https://give.stanford.edu/',
+  giving: 'https://www.giving.stanford.edu/',
+};
```

Comment:
Starting a file for any external links that might be used more than once (esp those that might need to be updated in the future, e.g., SNOW ticket links if any)